gms/gossiper.cc

```diff
@@ -1804,6 +1804,8 @@ future<> gossiper::do_shadow_round(std::unordered_set<gms::inet_address> nodes,
                 }).handle_exception_type([node, &fall_back_to_syn_msg] (seastar::rpc::unknown_verb_error&) {
                     logger.warn("Node {} does not support get_endpoint_states verb", node);
                     fall_back_to_syn_msg = true;
+                }).handle_exception_type([node, &nodes_down] (seastar::rpc::timeout_error&) {
+                    logger.warn("The get_endpoint_states verb to node {} was timeout", node);
```

Comment:
Shouldn't we go the same path on timeout as we do on closed_error below, that is increment `nodes_down`? In both cases we could not contact the node.

Reply:
Yes, in both case we could not talk to the node. But the difference is that with timeout_error we have a good chance talking to the node if we retry, since scylla process is running, just the network or scylla is slow. We get closed_error if scylla is not running on a node.

We count nodes_down to skip the shadow round mainly for the case when all nodes were shutdown and the first node restarts. 

So, it is better not to increment nodes_down here.